Traceback (most recent call last):
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_model.py", line 126, in <module>
    main(args.input_folder, args.graph_file, int(args.epochs), float(args.lamb))
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_model.py", line 80, in main
    glist, _ = dgl.load_graphs(os.path.join(input_folder, graph_file))
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 186, in load_graphs
    check_local_file_exists(filename)
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 40, in check_local_file_exists
    raise DGLError("File {} does not exist.".format(filename))
dgl._ffi.base.DGLError: File saved_graphs/spoken_digit/fixed/knn/dnn/phon_count/hetero_graph_149_3_299_3000.dgl does not exist.
Traceback (most recent call last):
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_link_pred_model.py", line 152, in <module>
    main(args.input_folder, args.graph_file, int(args.epochs), float(args.lamb))
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_link_pred_model.py", line 99, in main
    glist, _ = dgl.load_graphs(os.path.join(input_folder, graph_file))
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 186, in load_graphs
    check_local_file_exists(filename)
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 40, in check_local_file_exists
    raise DGLError("File {} does not exist.".format(filename))
dgl._ffi.base.DGLError: File saved_graphs/spoken_digit/fixed/knn/dnn/phon_count/hetero_graph_149_3_299_3000.dgl does not exist.
Traceback (most recent call last):
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_model.py", line 126, in <module>
    main(args.input_folder, args.graph_file, int(args.epochs), float(args.lamb))
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_model.py", line 80, in main
    glist, _ = dgl.load_graphs(os.path.join(input_folder, graph_file))
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 186, in load_graphs
    check_local_file_exists(filename)
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 40, in check_local_file_exists
    raise DGLError("File {} does not exist.".format(filename))
dgl._ffi.base.DGLError: File saved_graphs/spoken_digit/fixed/knn/fixed/phon_count/hetero_graph_149_3_299_3000.dgl does not exist.
Traceback (most recent call last):
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_link_pred_model.py", line 152, in <module>
    main(args.input_folder, args.graph_file, int(args.epochs), float(args.lamb))
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_link_pred_model.py", line 99, in main
    glist, _ = dgl.load_graphs(os.path.join(input_folder, graph_file))
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 186, in load_graphs
    check_local_file_exists(filename)
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 40, in check_local_file_exists
    raise DGLError("File {} does not exist.".format(filename))
dgl._ffi.base.DGLError: File saved_graphs/spoken_digit/fixed/knn/fixed/phon_count/hetero_graph_149_3_299_3000.dgl does not exist.
Traceback (most recent call last):
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_model.py", line 126, in <module>
    main(args.input_folder, args.graph_file, int(args.epochs), float(args.lamb))
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_model.py", line 80, in main
    glist, _ = dgl.load_graphs(os.path.join(input_folder, graph_file))
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 186, in load_graphs
    check_local_file_exists(filename)
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 40, in check_local_file_exists
    raise DGLError("File {} does not exist.".format(filename))
dgl._ffi.base.DGLError: File saved_graphs/spoken_digit/fixed/knn/dnn/phon_count/hetero_graph_149_3_299_3000.dgl does not exist.
Traceback (most recent call last):
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_link_pred_model.py", line 152, in <module>
    main(args.input_folder, args.graph_file, int(args.epochs), float(args.lamb))
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_link_pred_model.py", line 99, in main
    glist, _ = dgl.load_graphs(os.path.join(input_folder, graph_file))
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 186, in load_graphs
    check_local_file_exists(filename)
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 40, in check_local_file_exists
    raise DGLError("File {} does not exist.".format(filename))
dgl._ffi.base.DGLError: File saved_graphs/spoken_digit/fixed/knn/dnn/phon_count/hetero_graph_149_3_299_3000.dgl does not exist.
Traceback (most recent call last):
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_model.py", line 126, in <module>
    main(args.input_folder, args.graph_file, int(args.epochs), float(args.lamb))
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_model.py", line 80, in main
    glist, _ = dgl.load_graphs(os.path.join(input_folder, graph_file))
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 186, in load_graphs
    check_local_file_exists(filename)
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 40, in check_local_file_exists
    raise DGLError("File {} does not exist.".format(filename))
dgl._ffi.base.DGLError: File saved_graphs/spoken_digit/fixed/knn/fixed/phon_count/hetero_graph_149_3_299_3000.dgl does not exist.
Traceback (most recent call last):
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_link_pred_model.py", line 152, in <module>
    main(args.input_folder, args.graph_file, int(args.epochs), float(args.lamb))
  File "/home/yannick/Documents/Acoustic_Linguistic_GNN/gnn_heto_link_pred_model.py", line 99, in main
    glist, _ = dgl.load_graphs(os.path.join(input_folder, graph_file))
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 186, in load_graphs
    check_local_file_exists(filename)
  File "/home/yannick/.local/lib/python3.10/site-packages/dgl/data/graph_serialize.py", line 40, in check_local_file_exists
    raise DGLError("File {} does not exist.".format(filename))
dgl._ffi.base.DGLError: File saved_graphs/spoken_digit/fixed/knn/fixed/phon_count/hetero_graph_149_3_299_3000.dgl does not exist.
